recipe/yum_requirements.txt

```diff
@@ -0,0 +1,16 @@
+liburiparser-devel
```

Comment:
Is there something wrong with this format perhaps? Since it only seems to install the following packages:
```
Installing:
 ImageMagick                  x86_64  6.7.2.7-5.el6_8            updates  1.9 M
 dos2unix                     x86_64  3.1-37.el6                 base      16 k
 fop                          x86_64  0.95-4.4.el6               base     2.6 M
 libxslt                      x86_64  1.1.26-2.el6_3.1           base     452 k
```